backends/bmv2/bmv2stf.py

```diff
@@ -320,8 +320,10 @@ def __init__(self, jsonTable):
         self.key = TableKey()
         self.actions = {}
         for k in jsonTable["key"]:
-            name = k["name"]
-            if name is None:
+            name = ""
+            if "name" in k:
+                name = k["name"]
+            else:
```

Comment:
Given the documentation for the `name` and `target` keys of a table definition in the BMv2 JSON file that you can find here https://github.com/p4lang/behavioral-model/blob/main/docs/JSON_format.md, then search for the only occurrence of "name: this attribute", this change appears like a correct one to me, in order to handle a larger set of correct BMv2 JSON files.  Pinging @antoninbas to see if he has any comment on this change, but I will approve this change regardless, in case he is busy.